OpenAnatomyExport/OpenAnatomyExport.py

```diff
@@ -469,18 +469,19 @@ def addModelToRenderer(self, inputModelNode, outputModelNode, boostGouraudColor=
     outputPolyData.GetPointData().GetNormals().SetName("NORMAL")
     outputModelNode.SetAndObservePolyData(outputPolyData)
 
-    ras2lps = vtk.vtkMatrix4x4()
-    ras2lps.SetElement(0,0,-1)
-    ras2lps.SetElement(1,1,-1)
-    ras2lpsTransform = vtk.vtkTransform()
-    ras2lpsTransform.SetMatrix(ras2lps)
-    transformer = vtk.vtkTransformPolyDataFilter()
-    transformer.SetTransform(ras2lpsTransform)
-    transformer.SetInputConnection(outputModelNode.GetPolyDataConnection())
+    transformOutputModelToWorldRAS = vtk.vtkGeneralTransform()
+    transformOutputModelToWorldRAS.PostMultiply()
+    slicer.vtkMRMLTransformNode.GetTransformBetweenNodes(inputModelNode.GetParentTransformNode(), None, transformOutputModelToWorldRAS)
+    
+    transformOutputModelToWorldLPS = transformOutputModelToWorldRAS
```

Comment:
This doesn't really explain things. You could create a new vtkGeneralTransform instead just assigning a new name but that would be a bit overcomplicated,l. Probably using just the transformOutputModelToWorldLPS name and a good comment would be sufficient.